RoboJackets/robocup-software
# Vision receiver module

## Description
Refactor vision receiver into a new node and refactor to use boost::asio instead of a separate thread. ~Currently marked as draft until #1330 (which includes the Node base class) is merged.~

## Associated Issue
Issue #1329.

## Design Documents
[Link](https://docs.google.com/document/d/13LJ-HotI7wjp8G1xqDhcK5y4MAMSmWZyr55kbajW1qw/edit#heading=h.3vz8rotbuhas)

## Steps to test
This is a non-functional change. It needs QA to make sure it didn't break any of the following functionality:
 - Vision on actual field
 - Using specific vision channels on the field
